# Disables Site search Starter Pack
# Adds DDG to the default search engines.
#
Index: chromium-128.0.6613.137/components/omnibox/common/omnibox_features.cc
===================================================================
--- chromium-128.0.6613.137.orig/components/omnibox/common/omnibox_features.cc
+++ chromium-128.0.6613.137/components/omnibox/common/omnibox_features.cc
@@ -528,7 +528,7 @@ BASE_FEATURE(kSiteSearchSettingsPolicy,
 // Enables additional site search providers for the Site search Starter Pack.
 BASE_FEATURE(kStarterPackExpansion,
              "StarterPackExpansion",
-             enabled_by_default_desktop_only);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Enables an informational IPH message at the bottom of the Omnibox directing
 // users to certain starter pack engines.
Index: chromium-128.0.6613.137/components/search_engines/search_engine_countries-inc.cc
===================================================================
--- chromium-128.0.6613.137.orig/components/search_engines/search_engine_countries-inc.cc
+++ chromium-128.0.6613.137/components/search_engines/search_engine_countries-inc.cc
@@ -36,6 +36,7 @@ constexpr EngineAndTier engines_default[
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
+    {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Note, the below entries are sorted by country code, not the name in comment.
